site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Troubleshooting missing or orphan spans
+weight: 20
+menu:
+  platform:
+    parent: "Send your traces to Checkly"
+    identifier: troubleshooting-missing-spans
+navTitle: Missing spans
+beta: true
+---
+
+This guide will walk you through the process of identifying missing spans, understanding their root causes, and implementing solutions to ensure your OpenTelemetry setup with Checkly provides complete and accurate tracing information. 
+
+## Missing spans
+
+Missing spans, also known as orphan spans, are segments of a trace that are disconnected from either their parent spans or from the root span.
+
+An instrumented distributed system, will emit spans creating a hierarchical chain that follows the flow of the request through the different components and services.
+
+When a span is missing, they create a whole in this chain, making it difficult to diagnose issue and understand the full picture.
+
+When viewing trace details, a span might be missing in the Checkly visualization. 
+Using contextual span data, about the trace it belongs to, and it's parent span; Checkly displays any missing pieces on the span chain, so that you can identify which application or service might be incorrectly instrumented. 
```

Comment:
```suggestion
Checkly uses contextual span data—such as the trace it belongs to and its parent span—to display any missing pieces in the span chain. This helps you identify which application or service may be incorrectly instrumented.
```

Reply:
I tripped over this sentence initially - again just a suggestion to restructure it for clarity. Feel free to ignore :)